lib/ui/dashboard.js

```diff
@@ -781,25 +781,51 @@ class Dashboard {
       screen.render();
     }
 
-    setupToggle(screen, debug);
+    setupToggle(screen, debug, [pods_table, pod_log]);
   }
 }
 
-function setupToggle (screen, debug) {
-  const validElementValues = Object.values({
-    logs: "Logs",
-    pods: "Pods",
-  });
+function setupToggle (screen, debug, validElementValues) {
+  validElementValues.forEach( vE => {
+    let element = getElementWithLabel(vE.options.label);
+    if (element) {
+      element.on('focus', () => {
+        element.style.border.fg = 'red';
+      });
+      element.on('blur', () => {
+        element.set("selected", false);
+        element.style.border.fg = "white";
+      });
+    }
+  })
+
+  // screen.children.forEach(element => {
+  //   const childLabel = element.options.label;
+  //   validElementValues.find(e => e.options.l)
+
+  //   validElementValues.forEach(element => {
+  //     if (element.options.label === childLabel ) {
+  //     }
+  //   })
+  // });
 
   screen.key(['tab', 'S-tab'], function (_, key) {
-    const focused = screen.children.find(element => element.focused);
-    let currentIndex = validElementValues.indexOf(focused.options.label)
-    let nextIndex = getNextIndex(currentIndex, validElementValues.length - 1, (key.shift && key.name === 'tab') ? -1 : 1)
-    let target = getElementWithLabel(validElementValues[nextIndex])
-    
-    if (target) target.focus();
+    let currentIndex = getIndexOfFocused()
+    if (currentIndex >= 0) {
+      let nextIndex = getNextIndex(currentIndex, validElementValues.length - 1, (key.shift && key.name === 'tab') ? -1 : 1)
+      let target = getElementWithLabel(validElementValues[nextIndex].options.label)
```

Comment:
Can't this be removed, and `validElementValues[nextIndex]` used directly as target?